pkg/cpio/cpio.go

```diff
@@ -96,3 +96,9 @@ func (w Writer) WriteRecords(files []Record) error {
 func (w Writer) WriteTrailer() error {
 	return w.WriteRecord(TrailerRecord)
 }
+
+func MakeReproducible(files []Record) {
```

Comment:
here's the reason. Suppose we make a totally new initramfs.cpio from unchanged Go source for whatever reason, where the compilers are the same, the ref of the Go source and u-root are the same. You want the resulting initramfs to be bit-for-bit the same as any initramfs created at any other time --> i.e. reproducible build of the cpio. Even if the contents of initramfs.go are totally unchanged, the MTime of the file will change, and the cpio will not the same. Reproducibility requires that build artifacts not have stuff like datestamps in them. Further, the inumbers can't be derived from the file system, since that will have changed, hence the other change where we create a synthetic inumber, rather than use the inumber of the file. 

that way,two coreboot roms created the same way with the same coreboot and kernel commits should be bit for bit identical.

I'll add that comment, and thanks for the review!